helpers/audioflingerglue-localbuild.spec

```diff
@@ -9,7 +9,7 @@
 
 Name:          audioflingerglue
 Summary:       Android AudioFlinger glue library
-Version:       0.0.1
+Version:       0.0.0
```

Comment:
This would require a new release of the HADK document, unless you left this as 0.0.1

Reply:
Just wanted to be consistent. Shouldn't this enforce the requirement to get the git tag of the commit to get the pkg version?

Reply:
HADK 3.0.1 is just around the corner, so it will all be consistent now :)
I'm rewording your commit message a little, will push to your branch and merge. Thanks!

Reply:
Hm it doesn't allow me to push to your branch, even at the bottom of this PR is written "Add more commits by pushing to the audiofinger_localbuild_version branch on Thaodan/droid-hal-device."
Most probably it's because you forked not immediately from upstream. I'll merge it as-is :)